filters/ThirdParty/filter_255_Phishing_URL_Blocklist/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+  "filterId": 255,
+  "name": "Phishing URL Blocklist",
+  "description": "A list of phishing websites, compiled from OpenPhish and mitchellkrogza/Phishing.Database.",
```

Comment:
The way it's compiled is a subject to change so it would be better to use this description from the repo itself:
```
Phishing URL blocklist for uBlock Origin (uBO), AdGuard, Vivaldi, Pi-hole, Hosts file, Dnsmasq, BIND, Unbound, Snort and Suricata.
```